llm_on_ray/finetune/finetune_config.py

```diff
@@ -96,6 +99,8 @@ class Training(BaseModel):
     gradient_accumulation_steps: int = 1
     logging_steps: int = 10
     deepspeed_config_file: str = ""
+    use_dpo: Optional[bool] = False
```

Comment:
maybe we should add option like `finetuning_mode`, available value is 'sft', 'dpo', 'ppo'